src/CoreWCF.Http/src/CoreWCF/Security/SecurityUtils.cs

```diff
@@ -1,7 +1,9 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Net;
 using System.Security.Principal;
```

Comment:
You left behind a coupe of new usings when you backed out the code.

Reply:
Good catch, will search for any others